grade.go

```diff
@@ -401,40 +417,26 @@ func main() {
 
 		result.StudentId = hydratedStudent.Id
 
-		if language == "python3" || language == "python" {
+		if language == "python3" || language == "python" || language == "javascript" || language == "js" {
 			result.CompileSuccess = true
 
-			stdout := runInterpreted(filepath.Join(workDir, dir), runArgs, input)
-			fmt.Printf("Output for %s: %s", result.Student, stdout)
-			result.RunCorrect, result.Feedback = compare(expected, stdout)
+			stdout := runInterpreted(filepath.Join(workDir, dir), runArgs, language, input)
+			fmt.Printf("Output For %s: %s", result.Student, stdout)
+			result.RunCorrect, result.Feedback = processOutput(expected, stdout)
```

Comment:
since we're passing the `language` to `gradeSubmission` anyways, let's pass the logic over to that function

Reply:
That would make more sense now that you mention it.